integreat_cms/cms/models/users/user.py

```diff
@@ -110,6 +110,13 @@ class User(AbstractUser, AbstractBaseModel):
             "Will be set to true once the user dismissed the page tree tutorial"
         ),
     )
+    statistics_tutorial_seen = models.BooleanField(
```

Comment:
```suggestion
    has_seen_statistics_tutorial = models.BooleanField(
```
I was told that it's good practice to give boolean fields an according name that makes it clear, that this validates to yes/no or true/false :)

Reply:
I know your feeling :sweat_smile: 

However there is already `page_tree_tutorial_seen`, which has the same purpose and I'd like to stay consistent with the existing code (or we should open another PR for renaming `page_tree_tutorial_seen` or first merge this and rename the both together).

Reply:
#3271